src/app/api/job/[jobId]/results/route.ts

```diff
@@ -13,50 +13,40 @@ export const POST = wrapApiOrgAction(async (req: Request, { params }: { params:
         return new NextResponse('Unauthorized', { status: 401 })
     }
 
-    try {
-        const formData = await req.formData()
-        const contents = formData.get('file') as File
-
-        if (!contents) {
-            return NextResponse.json({ status: 'fail', error: 'no "file" entry in post data' }, { status: 500 })
-        }
-
-        // join is a security check to ensure the job is owned by the org
-        const info = await db
-            .selectFrom('studyJob')
-            .innerJoin('study', (join) =>
-                join.onRef('study.id', '=', 'studyJob.studyId').on('study.orgId', '=', org.id),
-            )
-            .select(['studyJob.id as studyJobId', 'studyId'])
-            .where('studyJob.id', '=', jobId)
-            .executeTakeFirst()
-
-        if (!info) {
-            return NextResponse.json({ status: 'fail', error: 'job not found' }, { status: 404 })
-        }
-
-        await storeStudyResultsFile(
-            {
-                ...info,
-                orgSlug: org.slug,
-                resultsType: 'ENCRYPTED',
-            },
-            contents,
-        )
-
-        await db
-            .insertInto('jobStatusChange')
-            .values({
-                status: 'RUN-COMPLETE',
-                studyJobId: info.studyJobId,
-            })
-            .execute()
-
-        await sendResultsReadyForReviewEmail(info.studyId)
-
-        return NextResponse.json({ status: 'success' }, { status: 200 })
-    } catch (e) {
-        console.error(e)
-        return NextResponse.json({ status: 'fail', error: e }, { status: 500 })
+    const formData = await req.formData()
+    const logs = formData.get('log')
+    const results = formData.get('result')
+
+    const contents = logs || results // TODO: handle both logs and results
+    if (!contents || !(contents instanceof File)) {
```

Comment:
@chrisbendel this is where you'll need to hook into in order to store the logs.  instead of assigning to `contents` and using only that file, you'll want to store each separately and set status depending on the presence of 'results'